imports/plugins/included/product-variant/components/productGrid.js

```diff
@@ -30,7 +30,8 @@ const styles = (theme) => ({
     paddingTop: theme.spacing(3)
   },
   filterCountText: {
-    paddingLeft: theme.spacing(2)
+    paddingLeft: theme.spacing(2),
+    fontWeight: 400
```

Comment:
We should always try to use one of our variables, instead of hardcoding any numbers in styles.

In this case, use `theme.typography.fontWeightRegular` here instead of hardcoding `400`.